openedx/core/djangoapps/auth_exchange/views.py

```diff
@@ -35,6 +35,12 @@ class AccessTokenExchangeBase(APIView):
     View for token exchange from 3rd party OAuth access token to 1st party
     OAuth access token.
     """
+    # Do not attempt to authenticate for the access token exchange.
+    # The request payload should have all that it needs, and
+    # SessionAuthentication can cause issues by doing a CSRF check and
+    # by setting the session cookie.
+    authentication_classes = []
+
     @method_decorator(csrf_exempt)
```

Comment:
This should be removed since it doesn't work and that's the whole reason we're updating the `authentication_classes`.

Reply:
We should also add a comment on the `get` to make it clear that we override it because the method signature of our call from dispatch differs from the one of the underlying API view `get` definition.